spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/utils/jackson/RegistrationDeserializer.java

```diff
@@ -40,26 +40,26 @@ public Registration deserialize(JsonParser p, DeserializationContext ctxt) throw
 		JsonNode node = p.readValueAsTree();
 		Registration.Builder builder = Registration.builder();
 
-		if (node.has("name")) {
+		if (node.hasNonNull("name")) {
```

Comment:
If fields with a null value deserialized then asText() results in "null". For that a check if field has a none null value is required.

Reply:
👍